core/src/main/java/tc/oc/pgm/controlpoint/ControlPoint.java

```diff
@@ -354,6 +354,17 @@ private void dominateAndFireEvents(@Nullable Competitor dominantTeam, Duration d
       this.match.callEvent(
           new ControllerChangeEvent(this.match, this, oldControllingTeam, this.controllingTeam));
 
+      ScoreMatchModule scoreMatchModule = this.getMatch().getModule(ScoreMatchModule.class);
+      // Gives a set number of owner points to a team when captured.
+      if (this.controllingTeam != null && scoreMatchModule != null) {
```

Comment:
```suggestion
      if (scoreMatchModule != null) {
```
If you do not apply this logic when the controlling team is null, capturing won't give points since the current controlling team is null. In general points with a netrual state won't have this feature working as intended